src/sync.c

```diff
@@ -181,12 +187,14 @@ void detect_cfo(sync_t *st)
         for (int i = 0; i <= PM_PARTITIONS; i++)
         {
             adjust_ref(st, cfo + LB_START + i * PARTITION_WIDTH, cfo);
-            offset = find_ref(st, cfo + LB_START + i * PARTITION_WIDTH, (PM_PARTITIONS-i) & 0x3);
+            offset = find_ref(st, cfo + LB_START + i * PARTITION_WIDTH, (30-i) & 0x3);
```

Comment:
In a few places you use the constant `30`, what is this referring to?

Reply:
This comes from table 11-3 (Reference Subcarrier Identification) in 1011s.pdf. The RSID bits are symmetric about column 30. So column 30 has an RSID of 0, columns 29 and 31 have an RSID of 1, column 28 and 32 have an RSID of 2, etc. I struggled to come up with a good name for this, but I'll think of something.

Reply:
Ah ok. It doesn't need to be a good name, just preferably something where you can add a comment like that.